src/metrics-middleware.js

```diff
@@ -120,6 +120,10 @@ function _getRoute(req) {
             const baseUrl = splittedUrl.slice(0, routeIndex).join('/');
             route = baseUrl + route;
         }
+
+        if (setupOptions.includeQueryParams === true && req.query && Object.keys(req.query).length > 0) {
```

Comment:
checking that req.query exists is redundant. see https://expressjs.com/en/api.html#req.query

Reply:
cool, we remove it